src/views/dashboards/Dashboard/index.vue

```diff
@@ -192,21 +200,33 @@ export default {
     font-size: $unnnic-font-size-body-lg;
   }
   &__widgets {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    gap: $unnnic-spacing-md;
     margin-top: $unnnic-spacing-sm;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(0, 1fr));
+    grid-template-rows: repeat(3, auto);
+    gap: $unnnic-spacing-md;
+    grid-template-areas:
+      'cards cards graph'
+      'cards cards agents'
+      'cards cards agents';
   }
   &__widgets-cards {
-    width: 100%;
+    grid-area: cards;
     display: grid;
-    grid-template-rows: 1fr 1fr 1fr;
-    grid-template-columns: 1fr 1fr;
-    grid-gap: $unnnic-spacing-sm;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, auto);
+    gap: $unnnic-spacing-sm;
+    grid-column: span 2;
+  }
+  &__widgets-chart {
+    height: 115px;
+    grid-area: graph;
+  }
+  &__widgets-agent {
+    grid-area: agents;
   }
   &__widgets-card {
-    height: -webkit-fill-available;
+    height: 128px;
```

Comment:
```suggestion
    height: 100%;
```

Reply:
I believe this and the other adjustments above will make the layout fit the screen. Can you check if this is what you would like to get? Without the fixes, widgets would have a fixed size, which is not ideal for a quality frontend.